src/vg/civcraft/mc/citadel/listener/BlockListener.java

```diff
@@ -575,6 +576,27 @@ public void interact(PlayerInteractEvent pie) {
         }
     }
     
+    @EventHandler
+    public void onPlayerInteractEvent(PlayerInteractEvent event) {
+        Block block = event.getClickedBlock();
+        if ((event.getAction() == Action.PHYSICAL) && (block != null) && (block instanceof PressureSensor)) {
```

Comment:
instanceof will return false if the object is null, so you dont need an additional null check here